File: Frontend/funda-report-app/src/app/models/MakelaarTableModel.ts
import { MakelaarRowModel } from "./MakelaarRowModel";

export interface MakelaarTableModel {
    query?: string;
    numberOfApiRequests?: number;
    totalTimePreparingTable?: number;
    totalTimeWaitingOnRateLimit?: number;
    rows?: MakelaarRowModel[];
}

File: FundaReport/Models/MakelaarTableModel.cs
namespace FundaReport.Models
{
    public class MakelaarTableModel
    {
        public string Query { get; set; }
        public int NumberOfApiRequests { get; set; }
        public int TotalTimePreparingTable { get; set; }
        public int TotalTimeWaitingOnRateLimit { get; set; }
        public List<MakelaarRowModel> Rows { get; set; }
    }
}
